doc/source/parameters/cfd/constrain_solid_domain.rst

```diff
@@ -0,0 +1,34 @@
+=======================
+Constrain Solid Domain
+=======================
+
+This subsection is used to define temperature-dependant solid domains within a defined fluid.
+Homogenous constraints are applied on velocity and pressure degrees of freedom of cells found within the prescribed temperature range to mimic a solid.
+
+The subsection with default parameters goes as follows:
+
+.. code-block:: text
+
+    subsection constrain solid domain
+      set enable                = false
+      set number of constraints = 1
+      subsection constraint 0
+        set fluid id        = 0
+        set min temperature = -999.0
+        set max temperature = 0.0
+      end
+    end
```

Comment:
Maybe we should add a warning that ht has to be turned on to use the constraint.

Reply:
Good idea!